docs/security/user-certs.md

```diff
@@ -140,10 +140,27 @@ export CIGETCERTOPTS="-i 'Fermi National Accelerator Laboratory'"
 Your VO may also provide specific instructions for how to best use this tool.
 Contact your VO support center for details.
 
-Finally, cigetcert has advanced options for things like selecting an auth method, loading a configuration from a server, or storing the cert on a MyProxy server.
+Finally, `cigetcert` has advanced options for things like selecting an auth method, loading a configuration from a server, or storing the cert on a MyProxy server.
```

Comment:
I didn't comment on this before but let's tighten up the language here and drop `things like`.